week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,17 @@
-// Add comments to explain what this function does. You're meant to use Google!
+/* Math.random() *10: returns a random number between 0 (included) and 1 (excluded) times 10 */
 function getNumber() {
     return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+/* this function concatenates three words together */
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
+    return firstWord.concat(" " + secondWord + " " + thirdWord);
```

Comment:
Your solution is correct, however, the method concat has the ability to do the concatenation without the + sign 
something like 
```firstWrod.concat(" ",secondWord, " ", thirdWord);```